dandiapi/api/zarr_checksums.py

```diff
@@ -3,7 +3,7 @@
 from contextlib import AbstractContextManager
 from dataclasses import dataclass, field
 from pathlib import Path
-from typing import TYPE_CHECKING, Optional
+from typing import TYPE_CHECKING, Mapping, Optional
```

Comment:
```suggestion
from collections.abc import Mapping
from typing import TYPE_CHECKING, Optional
```

`typing.Mapping` is deprecated now https://peps.python.org/pep-0585/#implementation

Reply:
TIL, cool